Path-Check/covid-safe-paths
# Feature/updated url filter (PR/791)

#### Description:

Continuation of the work began in #791 by @Mr-EZG

Adds URL validation to the input on `ChooseProvider` screen using the [`valid-url` npm package
](https://github.com/ogt/valid-url#methods)

Updates to `CONTRIBUTING.md` are default styling from prettier that was not applied previously.

#### Linked issues:

#456 

#### Screenshots:

<img width="389" alt="Screen Shot 2020-05-12 at 10 17 33 PM" src="https://user-images.githubusercontent.com/20157849/81767587-62b51680-949e-11ea-84f4-e8b07b413ce5.png">

#### How to test:

- Attempt to enter an invalid URL (e.g. blank string, `bad_string`, `test.co`)
- Enter a valid URL and confirm that it is still added to the state
